webapp/src/main/kotlin/webapp/ports/NotificationController.kt

```diff
@@ -0,0 +1,24 @@
+package webapp.ports
+
+import domain.actions.ChangeStatusAction
+import domain.actions.ChangeStatusRequest
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/v1/notification")
+class NotificationController(
+    private val changeStatusAction: ChangeStatusAction
+) {
+
+    @PostMapping("/change-status")
+    fun changeStatus(
+        @RequestBody request: ChangeStatusRequest
+    ): ResponseEntity<Any> {
+        changeStatusAction.changeStatus(request)
+        return ResponseEntity.noContent().build()
```

Comment:
MAJOR: can this fail? Is it fire and forget?

Reply:
This stuff is called by the embedded system, so I'm expecting a kind of fire and forget behaviour at least on that side (it's a sort of event also this API call)